### PR TITLE
Aplicar mudanças ao novo header.

### PR DESCRIPTION
**Pull request by thulioph**
**Link to task:**
- https://app.asana.com/0/1126980117378377/1128779768290467/f

**How to test:**
Run the application with `make dev` and it will be available at `http://localhost:8000/`.

**Screenshots:**
- [Smartphones](https://monosnap.com/file/ZizkZ9yMOrHUAkEZw1EHZ5OXBPJ25P)
- [Tablets](https://monosnap.com/file/UgQoT21IxAQTWHyFYgJau0jWZy6TSn)
- [Desktops](https://monosnap.com/file/c7deku1Qmw7MPqJY09tbd7gN6Rbg1m)

### CODE REVIEW

**Comment by lucianoratamero** [comment]
can you send us a screenshot?

**Comment by thulioph** [comment]
Hey, @lucianoratamero it was updated. Thanks!

File: themes/medius/scss/home.scss [hunk 1]
```diff
@@ -6,7 +6,7 @@
   justify-content: center;
 
   max-width: 680px;
-  margin: 20px auto;
+  margin: 0 auto 20px auto;
 }
```
Comment (lucianoratamero) [inline themes/medius/scss/home.scss]: in these cases, you may omit the last 'auto' ;]

Comment (thulioph) [reply themes/medius/scss/home.scss]: Cool, thanks!

**Pull request merged**